hooks/types.ts

```diff
@@ -91,8 +91,11 @@ export type Token = {
   logo_url: string;
 };
 
-export type EnhancedFormattedPool = Omit<FormattedPool, 'token0' | 'token1' | 'TVL'> & {
+export type EnhancedFormattedPool = Omit<FormattedPool, 'token0' | 'token1' | 'TVL' | 'volume0' | 'volume1' | 'cumulativeVolumeUSD'> & {
```

Comment:
`'token0' | 'token1' | 'TVL' | 'volume0' | 'volume1' | 'cumulativeVolumeUSD'` can be created seperately as an enum value and used here. We can have

```js
type something = 'token0' | 'token1' | 'TVL' | 'volume0' | 'volume1' | 'cumulativeVolumeUSD'

Omit<FormattedPool, something>
```

Just to make the code cleaner

Reply:
got it